### PR TITLE
fix windows open-url

### PR DESCRIPTION
**Pull request by Nothing-Works**
I believe I have fixed `open-url` feature on `Windows`. I have tested on both `Windows` and `Mac`. 
And this is related https://github.com/manojVivek/responsively-app/issues/155
Fix https://github.com/manojVivek/responsively-app/issues/155
Please let me know if I am missing something or something can be improved.

### CODE REVIEW

File: desktop-app/app/main.dev.js [hunk 1]
```diff
@@ -196,8 +199,9 @@ const createWindow = async () => {
     if (urlToOpen) {
       openUrl(urlToOpen);
       urlToOpen = null;
+    } else {
+      mainWindow.show();
```
Comment (Nothing-Works) [inline desktop-app/app/main.dev.js:203]: The reason why I add `else` is that there is `mainWindow.show();` in `openUrl()` and two `mainWindow.show();` calls is going throw an error on my machine.

File: desktop-app/app/main.dev.js [hunk 2]
```diff
@@ -90,6 +90,9 @@ const openUrl = url => {
  */
 
 app.on('will-finish-launching', () => {
+  if (process.platform === 'win32') {
+    urlToOpen = process.argv.filter(i => /^responsively/.test(i))[0];
+  }
```
Comment (Nothing-Works) [inline desktop-app/app/main.dev.js:95]: here I am checking if it's `Windows`, we are going to assign the `url` to `urlToOpen`.

**Comment by manojVivek** [comment]
@Nothing-Works Thanks for putting in the time and fixing it, really appreciate it! 🎉 

Just one clarification before merging this, now the extension opens the URL from browser both when the app is open and closed, right?

**Comment by Nothing-Works** [comment]
@manojVivek From what I have tested when the app is closed, it's going to open the app with the current browser URL.

If there is an open app:
 - `mac`: it's going to show the current `mainWindow` with correct URL
 - `windows`: it's going to open another electron instance with correct URL. And this was the previous behavior when you open the app multiple times on `Windows`.

If you want we can make it that there will be only one instance on `Windows` as well.

**Comment by manojVivek** [comment]
@Nothing-Works Yeah, I think, it makes more sense to open it in the `mainWindow` for windows as well.

**Comment by jjavierdguezas** [comment]
@manojVivek could the user decide via options?
- Open in current instance (if there is any)
- Open in new instance

**Comment by manojVivek** [comment]
I just that will keep it simpler, UX wise, and didn't want to confuse the user with more options. :) 

The user can anyways click on the `Back` button to go back to the URL that was open before this.

@jjavierdguezas  What do you think?

**Comment by jjavierdguezas** [comment]
> I just thought that will keep it simpler, UX wise, and didn't want to confuse the user with more options. :)
> 
> The user can anyways click on the `Back` button to go back to the URL that was open before this.
> 
> @jjavierdguezas What do you think?

yeah.. simpler is better and what you say makes sense

**Comment by manojVivek** [comment]
@Nothing-Works Thank you so much for staying committed and making the changes. Looking forward to more like this. Merging this in. 🎉

**Comment by manojVivek** [comment]
@all-contributors Please add @Nothing-Works for code.

**Comment by manojVivek** [comment]
@Nothing-Works Would you mind sharing your twitter handle? We would love to send a shout out on twitter for this work! 🙂

**Comment by Nothing-Works** [comment]
@manojVivek Hi I am very sorry that the last push was just a `rebase` as I wanted to get the latest updates from the master. The code was the same as before, which everything works but on `Windows` it is going to have another instance when you run the app. I am very sorry about this, should I have another that make it a single instance on `Windows`?

**Comment by Nothing-Works** [comment]
@manojVivek https://twitter.com/AndySong666 I will be very happy if you can do so, thank you very much.

**Comment by manojVivek** [comment]
> @manojVivek Hi I am very sorry that the last push was just a `rebase` as I wanted to get the latest updates from the master. The code was the same as before, which everything works but on `Windows` it is going to have another instance when you run the app. I am very sorry about this, should I have another pr that make it a single instance on `Windows`?

No worries, it will be great if you could do that. thanks! :)

**Pull request merged**